rmf.repos

```diff
@@ -0,0 +1,5 @@
+repositories:
+  rmf/rmf_demos:
+    type: git
+    url: https://github.com/open-rmf/rmf_demos.git
```

Comment:
I don't think we should clone `rmf_demos` here. If we need the tinyRobot model, suggest committing it to Fuel with the slotcar plugin. See my comment above on committing the .world and nav_graph files.

Reply:
Apart from the launch files that I guess we can duplicate, we need `rmf_demos_fleet_adapter` to manage the slotcar robots. Unless we copy it in (!) or create a much more bare simulation method we can't really get away from having `rmf_demos` cloned.
If we distributed at least `rmf_demos_fleet_adapter` on the buildfarm we could remove this dependency.

Reply:
Or we could also do the nuclear option of not using the `rmf_demos_fleet_adapter` but using the very old C++ `full_control` fleet adapter but that's also not a great idea imo

Reply:
What I have done for now is still clone it (since we need the fleet adapter) but remove all other `rmf_demos` dependencies.
Since the CI should do a build "up to" `nexus_integration_tests` the total impact should be only building one Python package which is negligible.
I am not sure a better option exists, other than writing a full mock fleet adapter with a mock simulation, using legacy fleet adapters or distributing binaries of the demo fleet adapter, as mentioned above.

Reply:
Fair enough. We should look into releasing `rmf_demos` binaries for Jazzy...